### ISSUE TITLE
State of active component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#592]
<dl><dt>Description</dt><dd> The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
In Dark/Light mode both. Color contrast of focus indicator lacks 3 to 1 contrast ratio. 
Element : 
1. With error ? Baz?
Foreground: #F4C5CE
Background: #FFFFFF
CC ratio:  1.5:1</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.</dd><dt>Source code</dt><dd><pre><code><select id="x_44_318" required="" aria-describedby="x_45_7829"><option class="Field__option" value="yes">yes</option><option class="Field__option" value="no">no</option></select></code></pre></dd><dt>Checkpoint</dt><dd>Non-Text Contrast - States of User Interface Components (1.4.11.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Select</dd></dl>

**Comment by schne324** [comment]
this has been fixed by the `<FieldWrap />` component (which increases contrast) see #498
